src/Stratis.Bitcoin/Consensus/ConsensusExceptions.cs

```diff
@@ -28,6 +28,14 @@ public ConnectHeaderException() : base()
         }
     }
 
+    /// <summary>Can validate PH yet.</summary>
+    public class InsufficientHeaderInformation : ConsensusException
```

Comment:
Lets rename this to `InsufficientHeaderInformationException` just so that we stick to convention ;)